src/browser/squirrel-update.coffee

```diff
@@ -21,6 +22,9 @@ fileKeyPath = 'HKCU\\Software\\Classes\\*\\shell\\Atom'
 directoryKeyPath = 'HKCU\\Software\\Classes\\directory\\shell\\Atom'
 backgroundKeyPath = 'HKCU\\Software\\Classes\\directory\\background\\shell\\Atom'
 environmentKeyPath = 'HKCU\\Environment'
+openWithKeyPath = 'HKCU\\Software\\Classes\\Atom'
+filetypeKeyPath = (fileExtension) -> "HKCU\\Software\\Classes\\.#{fileExtension}\\OpenWithProgIds"
+openWithFileExtensions = require path.join(binFolder, 'win32-open-file-extensions.json')
```

Comment:
Started moving this, but then thinking about it - it's not a platform specific file type, but it is a file for a specific platform.  There's also a .js file in the win folder.  .sh files as well (which are platform specific, but not Windows types).  Is the intent really to separate file types?  Will move it if desired, but wanted to ask the question.  If so, should it just go in the root of src or create a new subdirectory?